[New Content] - Overture Guides (#385)
overture-stack/website: markdown/documentation/guides/administration/introduction.md

---
title: Administration Guides
---

> **These guides are for** anyone seeking information on configuring and using an Overture platform. We will walk you through the essential information for configuring your Overture platform. 

Reviewer: rephrased for technical accuracy
```suggestion
**These guides are for** anyone seeking information on configuring and using a Overture as their platforms's foundation. We will walk you through the essential information for configuring it. 
```

> **You will need** docker installed. We recommend using Docker Desktop; for more information, visit [Dockers website](https://www.docker.com/products/docker-desktop/)

Reviewer: Docker desktop used not to work well in Windows, and since we cannot ensure people are using the latest versions, we want to avoid leading them in a complicated direction... 
Also, there are other alternative services that allow running these things, that are not made by Docker themselves.

```suggestion
**You will need** a docker service (or equivalent) installed. In most cases, we recommend using Docker Desktop; for more information, visit [Dockers website](https://www.docker.com/products/docker-desktop/)
```

Author: I will update all these statements now they should specify the version and resources

**Overview:** This guide covers basic administration tasks associated with setting up and maintaining an Overture platform. Topics covered include:

   - **[Updating the Data Model](/documentation/guides/administration/modelling/):** Learn how to update your data model to reflect your data requirements

   - **[Index Mappings](/documentation/guides/administration/indexmapping/):** Understand what Index mappings are and how to configure them

   - **[Search Portal Customization](/documentation/guides/administration/portalcustomization/):** Learn how to customize how data is displayed in your front-end data facets and table components

>  <Note title="Help us make our guides better">If you can't find what you're looking for please reach out to us on our Slack channel linked on the top right of your screen or by email at contact@overture.bio</Note>

Reviewer: "Slack channel" being `#general`? or are you referring to any other channel?
Asking because the link on the website connects to our "Slack team", where #general is the default channel.

Making sure this is what's intended here.

Author: I don't disagree, I can update this later though
